docs/user_manual/attachments.md

```diff
@@ -0,0 +1,88 @@
+attachments to a project using the Edit Project page. Find the Attachments section on the Edit page. In the toolbar at
+the top of the table, click "Upload." In the dialog that appears, choose "Upload new files." From here, you can browse
+and select the file or files you wish to attach. Multiple files can be selected by using the Ctrl and Shift keys.
+Select a category for the file(s), then click Upload to add the attachment(s).
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Adding Attachments to Multiple Items" %}
+
+If a file pertains to multiple samples or libraries, it can be uploaded once and attached to all of the related items.
+This saves both time and storage space, as the same file would be stored on the MISO server multiple times if you
+attached it to each item individually.
+
+From the Samples or Libraries list page, select the items you wish to attach a file to, then click the "Attach Files"
+button in the toolbar at the top of the table. In the dialog that appears, choose "Upload New Files," then browse and
+select the file or files you wish to attach, choose a category, and click Upload. Each of the selected files will be
+uploaded once and linked to all of the selected items.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Linking Project Attachments" %}
+
+Another way to share a file between multiple items is by linking a Project attachment. This involves first attaching a
+file to the project, and then linking it to one or more samples or libraries. This means that the file will be
+accessible from both the Edit Project page, and the Edit page for each item it was linked to. Again, the file is only
+stored on the MISO server once, but it is linked to all of the intended items.
+
+Before you can link the project file, you will need to attach the file to the project, as described above in
+[Adding Attachments to a Single Item](attachments#adding_attachments_to_a_single_item). You can then use either the
+same "Upload" button as when adding attachments to a single sample or library, or the same "Attach Files" button on the
+Samples or Libraries list page as when adding attachments to other items, depending on whether you want to link the
+attachment to one or multiple items. In the dialog, choose "Link Project File" and then select the file that you wish
+to link from the dropdown. Click the "Link" button to complete the process.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Downloading Attachments" %}
+
+The Attachments section of the Edit page for any item that allows attachments lists all of the files that are attached
+or linked to the item. To download any of these files, click its filename in the list.
```

Comment:
The first sentence is a bit hard to parse; maybe "Files can be downloaded on the Edit page of an item. The Attachments section on the Edit page lists all of the files that are attached or linked to the item. [...]"
(Mostly, I don't think you need to specify "for any item that allows attachments" and could perhaps get away with just deleting that and leaving the rest intact)